src/components/CollapsibleView.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+
+const propTypes = {
+    /** Main Content */
+    children: PropTypes.func.isRequired,
+
+    /** Whether the content is visible. */
+    isVisible: PropTypes.bool,
+};
+
+const defaultProps = {
+    isVisible: false,
+};
+
+const CollapsibleView = props => (
```

Comment:
`ToggleVisibilityView` maybe? Collapsible makes me think of things like this -> https://github.com/Eliav2/react-native-collapsible-view

Reply:
Yup, Thanks for the better name